modules/core/src/main/scala/gem/config/GcalConfig.scala

```diff
@@ -1,14 +1,16 @@
 package gem
 package config
 
-import gem.enum.{GcalArc, GcalContinuum, GcalShutter}
+import gem.enum.{GcalArc, GcalContinuum, GcalDiffuser, GcalFilter, GcalShutter}
+
+import java.time.Duration
 
 import scalaz._
 import Scalaz._
 
 import GcalConfig.GcalLamp
 
-case class GcalConfig(lamp: GcalLamp, shutter: GcalShutter) {
+case class GcalConfig(lamp: GcalLamp, filter: GcalFilter, diffuser: GcalDiffuser, shutter: GcalShutter, exposureTime: Duration, coadds: Int) {
```

Comment:
Is the `exposureTime` a property of the instrument configuration, or part of the step itself? Is it ever absent?